html-text/src/main/java/de/charlex/compose/HtmlText.kt

```diff
@@ -147,7 +148,8 @@ fun HtmlText(
     maxLines: Int = Int.MAX_VALUE,
     inlineContent: Map<String, InlineTextContent> = mapOf(),
     onTextLayout: (TextLayoutResult) -> Unit = {},
-    style: TextStyle = LocalTextStyle.current
+    style: TextStyle = LocalTextStyle.current,
+    onOpenUri: (label: String, uri: String, uriHandler: UriHandler) -> Unit = { _, uri, uriHandler -> uriHandler.openUri(uri)},
```

Comment:
Maybe better du remove UriHandler from callback and make the callback nullable with default as null. Within th function use `onLinkClicked?.invoke(annotatedString.substring(sa.start, sa.end), sa.item) ?: uriHandler.openUri(sa.item)`